models/java/gbfs-java-model/pom.xml

```diff
@@ -195,7 +179,7 @@
                     <dependency>
                         <groupId>org.mobilitydata.gbfs</groupId>
                         <artifactId>rule-factory</artifactId>
-                        <version>1.0.0</version>
+                        <version>1.0.2</version>
```

Comment:
Maybe we can use version ranges:
https://maven.apache.org/enforcer/enforcer-rules/versionRanges.html

Reply:
See https://github.com/MobilityData/gbfs-json-schema/pull/112#discussion_r1586486853